modules/ROOT/pages/lists.adoc

```diff
@@ -20,6 +20,22 @@ tinymce.init({
 });
 ----
 
+== Example adding the `listprops` menu item to the Tools menu
+
+The `listprops` xref:#menu-items[menu item] opens the List Properties dialog.
+
+This dialog only works with numbered lists. The menu item is inactive unless the selection or insertion point is within a numbered list.
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea",  // change this value according to your HTML
+  plugins: "lists",
+  toolbar: "bullist numlist",
+  menu: {tools: { title: 'Tools', items: 'listprops' }},
+});
```

Comment:
```suggestion
  toolbar: "bullist numlist",
  contextmenu: 'lists',
  menu: {tools: { title: 'Tools', items: 'listprops' }},
});
```
Were we also going to make a new context menu section for plugins with context menus?

Reply:
Are you referring to this page? https://www.tiny.cloud/docs/tinymce/6/editor-context-menu-identifiers/

Reply:
@MitchCTiny,

> Were we also going to make a new context menu section for plugins with context menus?

We are, but that docs-infrastructure-level change is planned for post-6.6.

Reply:
@brian-forte Should this be redone in a follow up ticket then?